qa/common/gen_qa_custom_ops

```diff
@@ -43,6 +43,8 @@ PYTORCH_IMAGE=${PYTORCH_IMAGE:=nvcr.io/nvidia/pytorch:$TRITON_VERSION-py3}
 
 CUDA_DEVICE=${RUNNER_ID:=0}
 
+[[ $RUNNER_GPUS =~ ^[0-9] ]] && DOCKER_GPU_ARGS=$(eval $NV_DOCKER_ARGS) || DOCKER_GPU_ARGS="--gpus device=$CUDA_DEVICE" 
```

Comment:
Can you explain this line please?

Reply:
`RUNNER_GPUS` custom variable that help us determine proper GPU type attached to the runner device. <br>
Without evaluating this variable we always picking up default `0` mapped device, which can be different from required GPU type or integrated CPU device.